### PR TITLE
captions 00:00 to 00:15

### PR DESCRIPTION
**Pull request by digitalronin**
Ignore the other PR - I forgot to create it against the appropriate branch.

### CODE REVIEW

File: 6/melinda-seckington-captions.srt [hunk 1]
```diff
@@ -201,16 +201,16 @@ MS: Yes.
 
 46
 00:02:53,030 --> 00:02:53,872
-[They both talk at once]
+But it's not a brand new rails app.
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:204]: Prefix this with "DN: "

File: 6/melinda-seckington-captions.srt [hunk 2]
```diff
@@ -201,16 +201,16 @@ MS: Yes.
...
 
 47
 00:02:53,870 --> 00:02:57,920
-DN: [inaud] it already has a red
-register route and a log in route.
+DN: It already has a register route
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:208]: Remove "DN: " prefix

File: 6/melinda-seckington-captions.srt [hunk 3]
```diff
@@ -513,7 +514,7 @@ DN: Oh that's...
 
 119
 00:08:21,072 --> 00:08:25,056
-MS: Yes, sublime, like soup, try to open
+MS: Yes, sublime, likes to, try to open...
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:517]: :+1:

**Comment by nelstrom** [comment]
Great job @digitalronin, thanks!

You can ignore my line notes. I'll make those amendments after merging this PR.

File: 6/melinda-seckington-captions.srt (hunk 3 above)
Comment (BonitaJane) [inline 6/melinda-seckington-captions.srt:517]: Ohhps - I thought soup might be wrong but then I do hear some strange words in the programming language!

**Comment by nelstrom** [comment]
Merged via cherry-pick at command line.

File: 6/melinda-seckington-captions.srt (hunk 1 above)
Comment (BonitaJane) [inline 6/melinda-seckington-captions.srt:204]: Noted and done.

File: 6/melinda-seckington-captions.srt (hunk 2 above)
Comment (BonitaJane) [inline 6/melinda-seckington-captions.srt:208]: Done

**Pull request closed without merging**